x-pack/plugins/enterprise_search/server/routes/app_search/sample_response_search.test.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { MockRouter, mockRequestHandler, mockDependencies } from '../../__mocks__';
+
+import { registerSampleResponseSearchRoute } from './sample_response_search';
```

Comment:
There was like, no great place to put this route, so I just created it's own file. I'm open to moving this elsewhere.

Reply:
Yeah I put the `/curation_search` endpoint under the curations routes too, so I would have no objections putting this under the `result_settings.ts` routes

Reply:
4a6ebdd63af84ec9a94290feb816f6c5cb1f8424